presto-verifier/src/main/java/com/facebook/presto/verifier/checksum/FloatingPointColumnValidator.java

```diff
@@ -158,14 +158,13 @@ public FloatingPointColumnValidator(VerifierConfig config)
         }
 
         // Use absolute error margin if either control sum or test sum is 0
-        if (controlSum == 0 || testSum == 0) {
-            double controlMean = controlSum / controlResult.getRowCount();
-            double testMean = testSum / controlResult.getRowCount();
-            double difference = abs(controlMean - testMean);
+        double controlMean = controlSum / controlResult.getRowCount();
```

Comment:
It is possible for controlResult.getRowCount() to return zero here? If not, perhaps, add a comment to explain why.

Reply:
Comment added
```
Row count won't be zero since otherwise controlSum and testSum will both be null, and this has already been handled above.
```